docs/new_assignment.rst

```diff
@@ -0,0 +1,57 @@
+Creating a new assignment template
+----------------------------------
+
+Creating a new assignment template involves getting your local assignment
+files into
+a new repository on GitHub that you can use as a template for a GitHub
+classroom assignment.
+
+To create a new assignment template called "assignment1" using abc-classroom::
+
+  $ abc-assignment_template -a assignment1
```

Comment:
ok i think i figured it out. there is no assignment flag. i just added the name and it worked but this needs to be run in a dir with a config file so i'll try to update the instructions now